fitbenchmarking/controllers/tests/test_controllers.py

```diff
@@ -90,11 +90,12 @@ def error_flags(self):
 
 
 class ControllerSharedTesting:
-    '''
+    """
```

Comment:
Is this a pylint thing? Or just preference/editor? I think I usually use single quote so I'd be keen to know if there's a reason not to!

Reply:
This was just a preference because the other classes in this file used double quotes rather than single